### ISSUE TITLE
Tilemap Collision not triggering Area2D body_entered properly in Mono

### ISSUE THREAD

**Issue by rafaelgdp** [godotengine/godot#36316]
**Godot version:** 3.2 stable Mono Version

**OS/device including version:** MacOS Catalina. MBP 2018.

**Issue description:** Tilemap collision not working properly when collision_use_kinematic is false. This is really weird as it was working before, but now it only works when I set collision_use_kinematic to true. In my view, static and kinematic collisions should trigger the body_entered Area2D signal interchangeably.

**Steps to reproduce:** Create a tilemap with a simple collidable block. Shoot an Area2D (like a bullet) that triggers a body_entered signal when it collides with a tilemap block. The body_entered signal is only triggered when the tilemap's collision_use_kinematic property is set to true.

**Minimal reproduction project:** N/A

**Comment by Calinou** [comment]
@rafaelgdp Please upload a minimal reproduction project to make this easier to troubleshoot.

Also, try reproducing the issue in GDScript to confirm whether it's related to C# or not.

**Comment by rafaelgdp** [comment]
> @rafaelgdp Please upload a minimal reproduction project to make this easier to troubleshoot.
> 
> Also, try reproducing the issue in GDScript to confirm whether it's related to Mono or not.

Hi, I tried to reproduce it again here on the latest stable version (3.2.1 stable) and I could not reproduce the issue again. I think it was fixed (or maybe it was my fault back then somehow). I tried fiddling with my own mono project and I created a GDScript minimal version too. Both of them worked as expected. Either way, I'm closing this issue. If I ever experience it again, I will post a minimal reproduction project!
